utils.py

```diff
@@ -27,14 +27,14 @@ def parse_arguments():
         action="store",
         default="uniform",
         type=str,
-        choices=["uniform", "kmeans"],
+        choices=["uniform", "kmeans", "orderless"],
         help="Row bucketing strategy.")
     parser.add_argument(
         "--col_bucketer",
         action="store",
         default="uniform",
         type=str,
-        choices=["uniform", "kmeans"],
+        choices=["uniform", "kmeans", "orderless"],
         help="Column bucketing strategy.")
```

Comment:
calling this orderless is very funny to me :).

Reply:
LOL.

Well, I was thinking of the matrix as being orderless, meaning I could arbitrarily re-order the data. So that's why I called it that. But yes, I agree it might be a confusing name!

Let's change it to "sorted" ?

TODO